fog/ledger/server/src/router_handlers.rs

```diff
@@ -246,7 +247,11 @@ where
     // If there's no new store and we don't have enough responses, decrement
     // remaining_retries and loop
     let mut remaining_retries = query_retries;
+    let _timer = ROUTER_QUERY_REQUESTS.start_timer();
     while remaining_retries > 0 {
+        if remaining_retries < query_retries {
```

Comment:
IIUC what this `if` statement is doing is only increasing the RETRIES counter if this is not the first iteration of the loop. I think it's more readable if its done after the `if + break` block down below.